app/src/test/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryControllerTest.kt

```diff
@@ -181,23 +182,24 @@ class CustomPrescriptionEntryControllerTest {
   @Test
   fun `when sheet is opened in edit mode and save is clicked after making changes, then the prescription should be updated`() {
     val prescribedDrug = TestData.prescription(uuid = prescriptionUuid, name = "Atnlol", dosage = "20mg")
+    val updatePrescribedDrug = prescribedDrug.copy(name = "Atenolol", dosage = "5mg")
```

Comment:
Should we rename this?
```suggestion
    val updatedPrescribedDrug = prescribedDrug.copy(name = "Atenolol", dosage = "5mg")
```